packages/lab/src/dialog/DialogContent.tsx

```diff
@@ -20,6 +20,11 @@ export const DialogContent = forwardRef<HTMLDivElement, DialogContentProps>(
   function DialogContent(props, ref) {
     const { children, className, ...rest } = props;
     const { dialogId } = useDialogContext();
+    const [scrollTop, setScrollTop] = useState(0);
+
+    const handleScroll = (e: React.UIEvent<HTMLElement>) => {
```

Comment:
Can we use full names please.
```suggestion
    const handleScroll = (event: React.UIEvent<HTMLElement>) => {
```